### DISCUSSION TITLE
Discusiones sobre el proyecto, aportes, ideas, etc

### DISCUSSION THREAD

**Discussion by zebnat** [zebnat/morpho-dimension#9] [Anuncios]
En este lugar si **se puede discutir y hablar en Español**

## 👋 ¡Bienvenidos!

Como veis, por todo el proyecto se recomienda la utilización del Inglés. Eso es solo para personas que quieran contribuir a nivel técnico en el proyecto.

Utilizaremos las discusiones para conectar con los miembros del proyecto y en general con la comunidad interesada en Morpho dimension.

Esparamos de tí que...

* Preguntes cosas que te preocupen.
* Compartas ideas relacionadas con Morpho D.
* Te comuniques con otros usuarios de Morpho D.
* Des la bienvenida a los demás y respetes a esta comunidad que contruimos juntos 💪.

## Tablero de cosas que se van a realizar en Morpho

Puedes echar un ojo al estado de cosas en que se están trabajando [aquí](https://github.com/zebnat/morpho-dimension/projects/1).

O entrando a [https://github.com/zebnat/morpho-dimension/projects/1](https://github.com/zebnat/morpho-dimension/projects/1)

### ¿Y ahora?
Si quieres, puedes empezar comentando aquí mismo introduciéndote a ti mismo. Dinos quien eres en tus respectivas redes sociales, incluída la red social de macrojuegos.

Recuerden. ¡utilizad esto sabiamente y evitad el mal uso!

**Comment by TheMiler** [comment]
Buenas gente,  mucho gusto! uwu. En macrojuegos soy miler1000

**Reply by zebnat** [reply]
¡Bienvenido!

**Comment by Sinder20** [comment]
Buenas gente, en macrojuegos soy mewicl y este es mi repositorio de GitHub :D saludos!

**Comment by Darkusito** [comment]
Buenas noches, soy Darkus.

_Darkus_Jaeger_ en Macrojuegos, nunca he usado esta página ni he colaborado en este tipo de cosas pero espero poder aportar algo. uwu

Edit : Con _ al inicio y al final de mi nickname, no sé por qué está en cursiva. xD

**Comment by zebnat** [comment]
@Darkusito @miler1000  Lo primero, sería conseguir que sepáis modificar archivos de un repositorio propio. Ya eso tiene cierta complejidad ya que debéis de algún modo conectar vuestra terminal con vuestra cuenta de github, y hay tres métodos para eso. https (obsoleto), ssh y github cli.

Después de eso, se podría probar a hacer fork a este repositorio (se clona este en el vuestro) y aprender a modificarlo allí, para luego hacer un `Pull Request` de los cambios e integrarlos aquí.

Es complicado, hace falta una guía que podría aprovechar para explicar en algún directo pero ando corto de tiempo ahora.

**Reply by TheMiler** [reply]
Pues me parece bien sobre hacer una guía y puedes hacerla cuando tengas tiempo. Nosotros somos pacientes. Gracias por explicarnos esto en un mensaje. 
Un saludo! 
Miler

**Reply by Darkusito** [reply]
Tú dime rana y yo salto. B)
Pero si, ocupo una guía para saber que hacer heh

**Comment by Pequiscinco** [comment]
Buenos días/tardes/ noches.

Soy px5 y quisiera aportar una idea para el Ranking De La Comunidad en Macrojuegos.com (obvio), donde se pueda ver el Rank que tuviste anteriormente, el Rank actual y la mejor posición que tuviste.

Ejemplo : 
 ..................._______________________________________________________________
 ....................\Puesto Anterior/ ........Puesto Actual/......Mejor Posición/   
 .................... \ . \......[200]........./ ...........\......[98]........../............\....[37].........../

Espero sea algo útil 😀 .

**Reply by Pequiscinco** [reply]
Se me bugueó el ejemplo y lo edité como 10 veces xD. Espero no se vea bugueado :d

**Reply by Darkusito** [reply]
Se ve bastante bien, y es una excelente idea.

**Reply by zebnat** [reply]
Hola ¿En qué parte crees que debería salir esa información?
Actualmente el ranking de comunidad en cierto modo muestra el puesto actual y el anterior, dado que muestra la diferencia respecto a tu puesto anterior.

La mejor posición de toda tu cuenta si que no se guarda, solamente como un logro. Ese dato si es de importancia podríamos hacer que se guarde aparte de los logros, por supuesto. Lo que me interesa es saber dónde visualizáis que salgan esas 3 informaciones, tened en cuenta que hay un límite en la interfaz gráfica.

**Reply by Darkusito** [reply]
Considero qué se vería bastante bien en el apartado de Logros, como una pequeña ventana que se desplega donde podemos ver a detalles el ranking de dicho usuario.

**Reply by Sinder20** [reply]
@zebnat se podría mostrar solo el mejor ranking, no necesariamente lo demás, ya que como mencionas el rango anterior y el actual ya se puede ver desde la pestalla comunidad, y también en el perfil sale tu rank actual, quizás en el cuadro de información solo agregar tu mejor rango y ya, una línea más de info no se vería mal jiji

**Comment by XimenaTiaret** [comment]
_**Muy buenas con todos, días, tardes, noches, bueno me llamo Ximena, y soy "SoyTixi" (xd) en macrojuegos, no sé mucho como se utiliza esto, pero quería presentarme, saludos.**_  **:D**

**Reply by zebnat** [reply]
Bienvenida @XimenaTiaret ! No es muy complicado, es como un foro.

**Reply by Lince31** [reply]
No sé ni que hago aquí XD.

**Comment by AngelHiramPR** [comment]
Hola! Me llamo Angel, soy el usuario AngelHiram (anteriormente A_n_g_e_l) me cree la cuenta hoy XD

**Comment by Lince31** [comment]
Ya me cree una cuenta, soy lince310.

**Comment by zZakiraZz** [comment]
algo que en lo personal me gustaria es poder cambiar tu nickname

**Comment by DarkPegassy** [comment]
Mucho gusto, soy ya de 5 casi 6 años en macrojuegos me presento me llamo angel1386 en macro y mi instagram personal es iva_van13.

**Comment by bxbycry** [comment]
Buenas noches a todos los presentes, Soy xjuliox un usuario ya algo antiguo y me llamó mucho la atención lo que vendría a ser el proyecto de **MD** Espero poder contribuir con mi granito de arena, así como todos los que están en el proyecto y poder ayudar a Zeb <3

**Comment by HyoZK** [comment]
hola! no soy tan buen programador pero en macrojuego me encuentro como "Kirito_Kun1" queria dejar alguna idea para cuando salga Morpho D. y es que se pudiera cambiar el nombre de usuario 1 sola vez cuando te creas la cuenta(ya que mas de uno se habra creado la cuenta a corta edad y ahora que son mayores no le agrada su nick), tambien un sistema de tradeo/regalo de items como puede ser objetos para los avatares o baterias de exp(para poder usarlo tener que cumplir unos requisitos para evitar multicuentas de tradeos).
